src/main/jhipster/src/main/webapp/app/shared/tree/tree.component.html

```diff
@@ -12,7 +12,7 @@ <h4 class="modal-title">{{title}}</h4>
                         (onNodeExpand)="onNodeExpand($event)"
                         selectionMode="multiple"
                         autoLayout="true"
-                        [(selection)]="selectedNodes"
+                        [(selection)]="this.selectedNodes"
```

Comment:
`this` can be omitted, right?

Reply:
Yes, I use it because it is clearer to me that it is a variable